src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -326,7 +326,7 @@ function TextArea({
       <textarea
         className={classnames(
           'border rounded p-2',
-          'text-color-text-light bg-grey-0',
+          'placeholder:text-grey-6 bg-grey-0',
```

Comment:
Inherit default text color (grey-9) and set a dark-enough placeholder color that has enough contrast with white backgrounds.